app/models/in_progress_form.rb

```diff
@@ -42,6 +42,8 @@ def cast(value)
   before_save :set_expires_at, unless: :skip_exipry_update
   after_save :log_hca_email_diff
 
+  has_one :form_submission, class: 'FormSubmission'
```

Comment:
this is just an assumption, since we are tracking responses in a different model

Reply:
A couple reasons why i prefer my pattern

- logically, an in progress form has one submission.  at that point the form is no longer 'in progress', making that a bad naming convention for the persistent source of truth
- Logically, I think a submission has multiple attempts
- and most of all, the InProgressForm record expires.  we need persistent source of truth to track what was submitted, so InProgressForm won't work for that

Reply:
i agree with that, name makes more sense.  will update